repl/tests/rexpect.rs

```diff
@@ -22,126 +19,137 @@ macro_rules! test {
     };
 }
 
+struct REPL {
+    session: PtySession,
+    prompt: &'static str,
+}
+
+impl REPL {
+    /// Defines the command, timeout, and prompt settings.
+    /// Wraps a rexpect::session::PtySession. expecting the prompt after launch.
+    fn new() -> Result<REPL> {
+        let command = "../target/debug/gluon -i";
+        let timeout: u64 = 10_000;
+        let prompt: &'static str = "> ";
```

Comment:
Perhaps the prompt should match on the regex `^> ` to make it a little bit more unlikely that one writes a flaky test? (It would still be possible to break by doing `io.println "> "` however so we would still need to be a little bit careful in the tests).

It should work well enough either way since any test can be written in such a way that it does not print out something that looks like the prompt.

Reply:
I wasn't able to make a regex behave as planned, but I agree that there should be a more robust option than the ">" string. It will definitely be good to keep that in mind when writing more tests.